libcloudforensics/providers/azure/internal/account.py

```diff
@@ -13,15 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Represents an Azure account."""
-
+import hashlib
 from time import sleep
-from typing import Optional, Dict
+from typing import Optional, Dict, Tuple, List
 
 # Pylint complains about the import but the library imports just fine,
 # so we can ignore the warning.
 # pylint: disable=import-error
+from azure.core.exceptions import ResourceExistsError
 from azure.mgmt.compute import ComputeManagementClient
 from azure.mgmt.compute.v2016_04_30_preview.models import DiskCreateOption
+from azure.mgmt.resource import SubscriptionClient
+from azure.mgmt.storage import StorageManagementClient
+from azure.storage.blob import BlobServiceClient
+from azure.mgmt.resource import ResourceManagementClient
```

Comment:
Please import modules or packages, not classes. https://google.github.io/styleguide/pyguide.html#22-imports